.swcrc

```diff
@@ -1,5 +1,13 @@
 {
   "$schema": "https://json.schemastore.org/swcrc",
+  // has to duplicate from package.json, see swc issue: https://swc.rs/docs/configuration/compilation#env
+  "env": {
+    "targets": {
+      "chrome": "112",
```

Comment:
could you use the percentage format and match what we have in our package.json?
also our browserslist could probably use updating.